### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by tanyacoffey123** [Montana-Media-Arts/120_CreativeCoding_Spring2019#1]
I'm a little confused as to what we are supposed to put in here. Just general questions about the the material?

**Comment by michaelcassens** [comment]
Yes, you are more than welcome to ask questions here.  This will allow myself or others to help out.  Hope this helps!

**Comment by tanyacoffey123** [comment]
I'm not sure how well I can do the CLI homework since I use the school computers for my homework. I will certainly try my best though.

**Comment by michaelcassens** [comment]
Well definitely keep me posted. We can adjust as necessary!

**Comment by constancedarlington13** [comment]
I believe I'm doing the issues part of the homework correctly by putting something in here?

**Comment by daylinscott** [comment]
> I believe I'm doing the issues part of the homework correctly by putting something in here?

yes good work, friend

**Comment by tanyacoffey123** [comment]
That is correct. We are responding to the "issue". 

Second response: Great job with the encouragement!

**Comment by tanyacoffey123** [comment]
I'm not really sure what I'm supposed to do as far as the CLI. We download that file and then what? Are we supposed to load it into DreamWeaver?

**Comment by jaredhamman** [comment]
> I believe I'm doing the issues part of the homework correctly by putting something in here?

That was my assumption as well, haha.

**Comment by jaredhamman** [comment]
> I'm not really sure what I'm supposed to do as far as the CLI. We download that file and then what? Are we supposed to load it into DreamWeaver?

My assumption is we're just supposed to get an idea of navigating our OS using commands rather than the GUI interface. (Example: Command prompt in Windows.)
At least that's what I gathered from it.

**Comment by Koernera45** [comment]
So are we supposed to put the issues here and on moodle?

**Comment by HaltonStancil** [comment]
Is anyone else doing/has done the HW-2 assignment?

**Comment by michaelcassens** [comment]
Hi there,

Please feel free to submit issues here and answer the forum question on Moodle.  This will be the last time I ask for a specific submission on GitHub issues. 

Hope this helps!

Michael

**Comment by michaelcassens** [comment]
Are you having questions on HW-2?  Please feel free to ask away!

**Comment by cerauno** [comment]
I have successfully found this forum?

**Comment by cerauno** [comment]
I think I'm too late, but I'm going to plow ahead irregardless.

**Comment by cerauno** [comment]
> I believe I'm doing the issues part of the homework correctly by putting something in here?

I believe so

**Comment by cerauno** [comment]
> yes good work, friend

I like your style, comrade

**Comment by Koernera45** [comment]
I am working on my game and I'm struggling to get my Sprite1 to react to my Sprite2. It only reacts sometime? I think Sprite1 reacts when it touches the Sprite2 in a certain way. Does anyone know how I can get them to react better? Or am I being too picky?

**Comment by michaelcassens** [comment]
Hi Ashley,

Would you mind sending your .sb3 file?  Thanks!

**Comment by Koernera45** [comment]
I emailed it to you! @michaelcassens
